blocks/editable/format-toolbar.js

```diff
@@ -141,12 +145,14 @@ class FormatToolbar extends Component {
 			} );
 		}
 
+		const isEditingLink = this.state.isEditingLink || this.state.linkValue === '#';
```

Comment:
This is the line causing the focus error. I can't understand why but if you drop the `|| this.state.linkValue === '#'` it will work